### DISCUSSION TITLE
Disable inline suggestions/hints for Markdown files

### DISCUSSION THREAD

**Discussion by binarylogic** [zed-industries/zed#17328] [Feature Requests]
Hi, I'm really loving Zed, but I've been unable to disable inline suggestions/hints for Markdown files. For example, the grey text here:

<img width="535" alt="Screenshot 2024-09-03 at 1 20 06 PM" src="https://github.com/user-attachments/assets/4cb59de1-0473-4769-a2b6-ca4ac5a3555e">

I find this extremely annoying. I have the following my settings:

```json
{
  "languages": {
    "Markdown": {
      "show_inline_completions": false
    }
  },
"inline_completions": {
    "disabled_globs": [".md", "docs/architecture.md"]
  }
}
```

Yet they still appear. What am I missing? Thank you for making such an excellent editor!

**Comment by SomeoneToIgnore** [comment]
The main thing you miss is Zed's version you use.

https://github.com/zed-industries/zed/pull/17104 had fixed that presumably, and should be available in Zed Preview 0.151.
Can you try that version?

**Comment by Stan-Stani** [comment]
If you're looking for how to disable the intellisense suggestions: https://github.com/zed-industries/zed/discussions/24416

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
